samples/ChatSample/Hubs/Chat.cs

```diff
@@ -5,31 +5,38 @@
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.SignalR;
+using System.Collections.Generic;
+using System.Collections.Concurrent;
 
 namespace ChatSample.Hubs
 {
-    // TODO: Make this work
     [Authorize]
     public class Chat : Hub
     {
+        private static readonly IDictionary<string, string> usersOnline = new ConcurrentDictionary<string, string>();
```

Comment:
wow. Is there any logic that can check different servers connections or hubs? Or maybe you have some ideas or can point me to some code to understand how this issue can be resolved?